test/test_jit_fuser.py

```diff
@@ -60,6 +62,21 @@ def func(x):
     def test_abs_cpu(self):
         self._test_fused_abs()
 
+    @unittest.skipIf(IS_SANDCASTLE, "NYI: fuser CPU support for Sandcastle")
+    @enable_cpu_fuser
+    def test_abs_cpu_unicode_temp_dir(self):
+        if IS_FILESYSTEM_UTF8_ENCODING:
+            with TemporaryDirectoryName(suffix='中文') as dname:
+                shell_env = os.environ.copy()
+                shell_env['TMP'] = dname
+                cmd = [sys.executable, os.path.basename(__file__), 'TestFuser.test_abs_cpu']
+                legacy_jit_flag = '--jit_executor=legacy'
+                for v in sys.argv:
+                    if v == legacy_jit_flag:
```

Comment:
This is bad. I don't know how to do this correctly without breaking other things.

Reply:
It is not that bad. A slightly better way is to use `multiprocessing`, but it is way more complex here. If you want to do that, you may write sth like the following.
```py
import multiprocessing as mp

def test_abs_cpu_unicode_temp_dir(self):
    ### It is a bit tricky in the next line that `xxx` has to be a global function instead of a class function.
    pre_tmp_value = os.environ.get("TMP", None)
    os.environ["TMP"] = "中文路径"
    p = mp.Process(target=xxx)
    p.start()
    os.environ["TMP"] = pre_tmp_value
    p.join()
    self.assertEqual(p.exitcode, 0)
```

Reply:
I don't like the idea of messing the original structures of the test cases so I didn't use multiprocessing. That should also be a valid way.

Reply:
Another way to do this is just make a hermetic Python file (not a full test script) to run the test in question. There are a few examples of this in the test suite currently

Reply:
I did give it a way, but I found it tedious to create another file for this case. This is one of those cases we know it should work (the `A` to `W` conversion). I think we can leave this as it is for now.